build.gradle

```diff
@@ -30,7 +30,7 @@ ext {
 
     supportLibVersion = '28.0.0'
     playServicesVersion = '15.0.1'
-    workVersion = '1.0.0-alpha11'
+    workVersion = '1.0.0-beta03'
```

Comment:
beta04 is available now for WorkManager
https://developer.android.com/jetpack/androidx/releases/work#1.0.0-beta04

Reply:
Beta05 is now also available: https://developer.android.com/jetpack/androidx/releases/work#1.0.0-beta05